docs/core/stories.rst

```diff
@@ -28,8 +28,43 @@ This is a guide to the story data format.
 Format
 ------
 
+With Rasa Open Source 2.0 release we've introduced a YAML format as unified and the most extendable way to manage the training data.
+With YAML users are free to distribute training data among any number of YAML files.
+
+Rasa Open Source YAML format file should specify the `version` key in order to be parsed correctly:
+
+- Training data files with no "version" specified will be considered to have the "latest" format version.
+- Training data files with a version greater that currently used Rasa Open Source will be skipped.
+
+
 Here's an example of a dialogue in the Rasa story format:
 
+In YAML format:
+.. code-block:: yaml
+
+    version: "2.0"
+    stories:
+    - story: greet + location/price + cuisine + num people  # name of the story - just for debugging
+      steps:
+      - intent: greet
+      - action: action_ask_howcanhelp
+      - intent: inform
+        entities:
+        - location: "rome"
+        - price: "cheap"  # user utterance
+      - action: action_on_it
+      - action: action_ask_cuisine
+      - intent: inform
+        entities:
+        - cuisine: "spanish"
+      - action: action_ask_numpeople  # action that the bot should execute
+      - intent: inform
+        entities:
+        - people: "six"
+      - action: action_ack_dosearch
+
+Or in Markdown format:
```

Comment:
I think the conclusion was to remove the documentation about the markdown format completely: it will still be available in the 1.10.x docs and we want to drive users to use yaml instead of having the choice to use either one of them

Reply:
I'd rather add a note ala "there is also a deprecated file format, you can find more information about that here: legacy-docs.rasa.com"

Reply:
agreed, though not sure where we point them to at the moment because right now it's not at `legacy-docs.rasa.com` but on the same page just a different version